rerun_py/rerun_sdk/rerun/_send_columns.py

```diff
@@ -102,7 +102,7 @@ def __init__(
         elif timestamp is not None:
             # TODO(zehiko) add back timezone support (#9310)
             self.type = pa.timestamp("ns")
-            self.times = [to_nanos_since_epoch(timestamp) for timestamp in timestamp]
+            self.times = [to_nanos_since_epoch(timestamp).astype("datetime64[ns]") for timestamp in timestamp]
```

Comment:
reading this now it strikes me we should check for numpy-arrays here and in those cases do array operations instead of slow-as-turtles looping 😬  Perhaps leave it as a TODO…

Reply:
Eh, can just do that real quick